packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -405,7 +405,7 @@ const PRESETS = [
         name: 'Chakra Storefront Hybrid Test Private SLAS Project',
         description: '',
         templateSource: {
-            type: TEMPLATE_SOURCE_NPM,
+            type: TEMPLATE_SOURCE_BUNDLE,
             id: '@salesforce/chakra-storefront'
```

Comment:
Are we not removing @salesforce for this and the next one?

Reply:
Oh wow, that `id:` should indeed be changed as well, will fix

Reply:
Fix is pushed